src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,34 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
-
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { AccordionProps, Accordion } from "./Accordion";
+
+describe("Accordion", () => {
+  
+
+  const renderComponent = (addtionalProps?: Partial<AccordionProps>) => {
+    const defaultProps : AccordionProps = {
+    title: "Click me",
+      children:  "testing",
+    };
+
+    render(<Accordion {...defaultProps} {...addtionalProps} />)
+};
+    it("renders without error", () => {
+        renderComponent();
+        expect(screen).not.toBeNull(); 
+      });
+
+      it('renders the button with the provided text', () => {
+        const buttonText = 'Click me';
+        
+        render(<Accordion title={buttonText}><span>testing1234</span> </Accordion>);
+        const buttonElement = screen.getByText(buttonText);
+        expect(buttonElement).toBeInTheDocument();
+        fireEvent.click(buttonElement);
```

Comment:
I left userEvent as an imported package as it is superior to `fireEvent`. `userEvent` mimics the user interactions with the UI, which is what we're always trying to test. `fireEvent` directly interacts with the dom, which is not what we want to test.